dev-haskell/hspec-core/hspec-core-2.7.4.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Is there a reason for this version as well as `2.7.5`? If not I'll remove it after PR is merged.

Reply:
Good catch, I took out `2.7.4`